README.md

```diff
@@ -184,8 +184,9 @@ you based on the suites you ask it to load._
 # Configuration
 
 The `wct` command line tool will pick up custom configuration from a
-`wct.conf.json` file located in the root of your project. It should export the
-custom configuration:
+`wct.conf.json` file located in the root of your project.  
+Or you can specify your own file via the commandline option: --configFile my.wct.conf.js  
```

Comment:
nit: can you include the backticks to format: `--configFile my.wct.conf.js` 

Also, would prefer wording this as an example:

> Or, you can specify your own file via the `--configFile` command line option. Example: `--configFile my.wct.conf.js`